tests/fixtures/browser.ts

```diff
@@ -34,10 +34,9 @@ import {
   getDefaultRegularTrackSettings
 } from 'src/content/app/genome-browser/state/track-settings/trackSettingsConstants';
 
-import type { CogList } from 'src/content/app/genome-browser/Browser';
-
 import type { ChrLocation } from 'src/content/app/genome-browser/state/browser-general/browserGeneralSlice';
 import type { RegionValidationResponse } from 'src/content/app/genome-browser/helpers/browserHelper';
+import type { CogList } from 'src/content/app/genome-browser/components/browser-cog/BrowserCogList';
```

Comment:
You shouldn't need anything cog-related in the browser fixture anymore. Check with the dev branch.

Reply:
It's not there anymore. However there was some cog related code in there which I removed in the latest commit.

Reply:
✅